example-auth/src/components/footer.tsx

```diff
@@ -0,0 +1,13 @@
+import packageJson from "../../package.json"
+
+export default function Footer() {
+  return (
+    <footer>
+      <p>Example built by</p>
+      <a href="https://ticianomorvan.me" target="_blank" rel="noreferrer">
+        Ticiano Morvan
+      </a>
```

Comment:
I don't think we should add this. We're moving towards removing all mentions of authors in our codebase as Appwrite is built by all contributors.